packages/asset/package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "coverage": "hardhat coverage --testfiles 'test/*.ts''test/*.js'",
     "node": "hardhat node --no-deploy",
+    "compile": "hardhat compile",
     "deploy": "hardhat deploy",
```

Comment:
We should remove this when we've moved all the asset deploy scripts to `packages/deploy` and changed test fixtures so they don't rely on deploy scripts